integration/spark/src/test/scala/org/apache/carbondata/view/rewrite/MVFilterAndJoinTest.scala

```diff
@@ -29,9 +29,11 @@ class MVFilterAndJoinTest extends QueryTest with BeforeAndAfterAll {
     sql("create table main_table (name string,age int,height int) STORED AS carbondata")
     sql("create table dim_table (name string,age int,height int) STORED AS carbondata")
     sql("create table sdr_table (name varchar(20),score int) STORED AS carbondata")
+    sql("set carbon.enable.mv = true")
   }
 
   def drop() {
+    sql("set carbon.enable.mv = false")
```

Comment:
please move this to afterALL method. please check and handle in other changed files as well